pySDC/projects/Second_orderSDC/dampedharmonic_oscillator_run_stability.py

```diff
@@ -1,65 +1,55 @@
 import numpy as np
-
 from pySDC.implementations.problem_classes.HarmonicOscillator import harmonic_oscillator
 from pySDC.implementations.sweeper_classes.boris_2nd_order import boris_2nd_order
 from pySDC.projects.Second_orderSDC.penningtrap_Simulation import Stability_implementation
 
 
 def dampedharmonic_oscillator_params():
     """
-    Runtine to compute modulues of the stability function
+    Routine to compute modules of the stability function
```

Comment:
Very misleading comment. This function supplies parameters. It doesn't compute anything.